api/pkg/apis/v1alpha1/providers/stage/http/http.go

```diff
@@ -214,6 +214,7 @@ func (i *HttpStageProvider) Process(ctx context.Context, mgrContext contexts.Man
 				return nil, false, err
 			}
 			req.Body = ioutil.NopCloser(bytes.NewBuffer(jData))
+			req.Header.Set("Content-Type", "application/json; charset=UTF-8")
```

Comment:
Is this set as configurable input?

Reply:
In the future, we might consider to support XML/other types.